src/lightning/fabric/wrappers.py

```diff
@@ -169,6 +171,18 @@ def load_state_dict(  # type: ignore[override]
     ) -> _IncompatibleKeys:
         return self._original_module.load_state_dict(state_dict=state_dict, strict=strict, **kwargs)
 
+    def mark_forward_method(self, method: Union[MethodType, str]) -> None:
+        """Mark a method as a 'forward' method to prevent it bypassing the strategy wrapper (e.g., DDP)."""
+        name = method if isinstance(method, str) else method.__name__
```

Comment:
edge case: Can you add a test for passing a function with the same name as an existing model method?